examples/full-example/.mendelrc

```diff
@@ -17,8 +17,8 @@ variation-config:
       - partner_C
 
 route-config: # Mendel-middleware related
-  variation: /mendel/:variations/:bundle # Route pattern for variational assets during development
-  hash: /mendel/:hash/:bundle # Route pattern for variational assets during production
+  variation: /mendel/:variations/:bundle.js # Route pattern for variational assets during development
```

Comment:
Outlet simply transform "bundle" object into something. This route business is an applicational concept that does not belong in outlet. Yeah i wondered why we dont use express. 

Random thought: multiple mendel clients per express app feels bad. :(